sympy/integrals/transforms.py

```diff
@@ -1150,9 +1175,193 @@ class LaplaceTransform(IntegralTransform):
 
     _name = 'Laplace'
 
+    def _cr(self, f, a, c, **hints):
+        conds = not hints.get('noconds', False)
+        if conds:
+            return f, a, c
+        else:
+            return f
+
     def _compute_transform(self, f, t, s, **hints):
+        a = Wild('a', exclude=[t])
+        b = Wild('b', exclude=[t])
+        n = Wild('n', exclude=[t])
+        g = WildFunction('g', nargs=1)
+        tau = Wild('tau', exclude=[t])
+        omega = Wild('omega', exclude=[t])
+        dco = lambda f: self._deep_collect(f,t)
+        if not any(func.has(t) for func in f.atoms(AppliedUndef)):
+            simple_rules = [
+                # ( time domain, laplace domain,
+                #       condition, convergence plane, preparation function )
```

Comment:
It would be better to define all these rules as a global variable in the module I think.

Reply:
Or maybe not because then it would happen at import time.

Either way there should be a way to get these rules directly i.e. a function that can be called like `_get_laplace_transform_rules()`.

Reply:
agreed, will do

Reply:
Just a question: that function will need all the wildcards. Should these be passed as arguments?

Reply:
I think the function should return the wildcards.

Reply:
Do users of the function need them at all, or can they just be part of the returned expressions?

Reply:
I have written something that should be useful.